### PR TITLE
add null check for enqueuedTime in EventDataConsumer, ignore resource_leak in DefaultOutput.java.

### PR DESCRIPTION
**Pull request by eemhu**
fix coverity issue

### CODE REVIEW

File: src/main/java/com/teragrep/aer_02/EventDataConsumer.java [hunk 1]
```diff
@@ -146,7 +146,7 @@ public Long getValue() {
         SyslogMessage syslogMessage = new SyslogMessage()
                 .withSeverity(Severity.INFORMATIONAL)
                 .withFacility(Facility.LOCAL0)
-                .withTimestamp(enqueuedTime.toInstant())
+                .withTimestamp(enqueuedTime == null ? ZonedDateTime.now().toInstant() : enqueuedTime.toInstant())
```
Comment (MoonBow-1) [inline src/main/java/com/teragrep/aer_02/EventDataConsumer.java:149]: Does the `ZonedDateTime.now()` provide the system time zone, or should it be locked to a single time zone?

**Review by MoonBow-1 — COMMENTED** [review]
A quick question about the intended behavior

File: src/main/java/com/teragrep/aer_02/EventDataConsumer.java (hunk 1 above)
Comment (eemhu) [reply src/main/java/com/teragrep/aer_02/EventDataConsumer.java:149]: It does indeed use the system timezone, and provided that EnqueuedTime is in UTC, the alternative time should also be in UTC. Changing to Instant.now().

File: src/main/java/com/teragrep/aer_02/EventDataConsumer.java [hunk 2]
```diff
@@ -146,7 +146,7 @@ public Long getValue() {
         SyslogMessage syslogMessage = new SyslogMessage()
                 .withSeverity(Severity.INFORMATIONAL)
                 .withFacility(Facility.LOCAL0)
-                .withTimestamp(enqueuedTime.toInstant())
+                .withTimestamp(enqueuedTime == null ? Instant.now() : enqueuedTime.toInstant())
```
Comment (kortemik) [inline src/main/java/com/teragrep/aer_02/EventDataConsumer.java:149]: please add structured data indicating that the timestamp was sourced from this component i.e. aer_02@48577 timestamp_source=generated/time enqueued

**Review by kortemik — CHANGES_REQUESTED** [review]

File: src/main/java/com/teragrep/aer_02/EventDataConsumer.java (hunk 2 above)
Comment (eemhu) [reply src/main/java/com/teragrep/aer_02/EventDataConsumer.java:152]: added

**Review by kortemik — APPROVED** [review]

**Pull request merged**